test/qa/test_meshnet.py

```diff
@@ -88,31 +88,218 @@ def test_mesh_removed_machine_by_other():
         assert "Meshnet is not enabled." in str(e)
 
     sh.nordvpn.set.meshnet.on() # enable back on for other tests
+    meshnet.add_peer(ssh_client)
+    sh.nordvpn.meshnet.peer.refresh()
```

Comment:
wouldn't work to have the `refresh` function called by the `add_peer` automatically, instead of writing it after each call?